app/SmartFlowUI/backend/Services/Profile/ProfileService.cs

```diff
@@ -0,0 +1,88 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+namespace MinimalApi.Services.Profile;
+
+public static class ProfileService
```

Comment:
can this be a non-static class that uses async and dependency injection? would be easier to test

Reply:
Probably could...  good change for another day... ;)